storage/swift.go

```diff
@@ -122,49 +120,67 @@ func (sw *SwiftBackend) get(src io.Reader, hostPath string) error {
 
 // PutFile copies an object (file) from the host path to storage.
 func (sw *SwiftBackend) PutFile(ctx context.Context, rawurl string, hostPath string) error {
-	url, perr := sw.parse(rawurl)
-	if perr != nil {
-		return perr
-	}
+	url := sw.parse(rawurl)
 
 	reader, oerr := os.Open(hostPath)
 	if oerr != nil {
 		return oerr
 	}
 
-	writer, err := sw.conn.ObjectCreate(url.bucket, url.path, true, "", "", nil)
+	var writer io.WriteCloser
+	var err error
+
+	// TODO turn on checkHash
```

Comment:
Yes, please do turn this on. If I understand correctly, it was previously on and was critical for preventing corrupted files in smc-het.